backstop/scripts/ready.js

```diff
@@ -49,6 +49,10 @@ module.exports = async function (page, scenario, vp) {
   await page.evaluate(function (url) {
     // Disable jQuery animation for any future calls.
     jQuery.fx.off = true;
+
+    // Zero delay on CSS transitions.
+    jQuery("head").append('<style type="text/css"> *, *:before, *:after { transition-duration: 0s !important } </style>');
```

Comment:
I like this because it would reduce all backstop test times, and we're not trying to test animations themselves.

However, do we still need to add a selector wait for the menu itself? It seems like there could still be a delay between calling click() and when the menu has _actually_ rendered.

Reply:
@deviantintegral calling a click and rendering should be executed synchronously, in the same thread right? The same for taking the screenshot later. I don't think we need to wait, I can't understand why we would need to wait. If you think waiting is necessary, I can add a small `wait` right now to PR. Thanks!

Reply:
It's the same thread, but all events are async AFAIK: https://stackoverflow.com/a/69111229/7982687

Regardless, if this is a real-world problem we'll see it pretty quickly and know what to do. Let's get this in and we can always do a followup.